packages/flutter_availability/lib/src/ui/view_models/break_view_model.dart

```diff
@@ -33,6 +33,14 @@ class BreakViewModel {
   /// [endTime]
   final Duration? duration;
 
+  /// Get the duration in minutes
+  /// If the duration is null, return the difference between the start and end
+  /// time in minutes
+  int get durationInMinutes =>
```

Comment:
There is a similar method on the BreakModel, maybe you could check that out. This is a bit too complex for the goal you try to achieve

Reply:
i now that method is there but would you want to convert the breakViewModel to a AvailabilityBreakModel to call that method on it?
like int get durationInMinutes => this.toBreak().duration.inMinutes?